test/e2e/nutanix_client_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	infrav1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
 )
 
-var _ = Describe("Nutanix client", func() {
+var _ = Describe("Nutanix client", Label("capx-feature-test", "slow", "network"), func() {
```

Comment:
Please add a specific label so we can target the specific client tests

Reply:
added